torch/onnx/_internal/exporter/_compat.py

```diff
@@ -32,6 +32,7 @@ def _from_dynamic_axes_to_dynamic_shapes(
     model,
     dynamic_axes=None,
     input_names: Sequence[str] | None = None,
+    output_names: Sequence[str] | None = None,
```

Comment:
```suggestion
    output_names: set[str],
```

Reply:
I thought output_names is Sequnce[str] just like input_names?

Reply:
I see.